benchmarks/Program.cs

```diff
@@ -0,0 +1,6 @@
+// See https://aka.ms/new-console-template for more information
```

Comment:
Missing license header. Am completely happy for this to be the the new project template 💯